src/test/java/com/test/Flipkart1.java

```diff
@@ -32,6 +29,12 @@ public void test2() {
         Assert.assertTrue(b);
     }
 
+    @Test
+    public void test3() throws InterruptedException {
+        Boolean b = getDetails2();
+        Assert.assertTrue(b);
```

Comment:
Instead of use two lines, merge into a single line :
Assert.assertTrue(getDetails2);